onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -130,4 +130,17 @@ def optimize_for_ort(
     )
     # Apply the ORT pattern rewrite rules.
     rewrite(model, ORT_PATTERN_REWRITE_RULES)
+
+    # TODO(exporter team): Fold transpose into initializers
+    # Apply the ORT optimization passes.
+    # https://github.com/microsoft/onnxruntime/blob/74dcf7e296639095dfa55d31336998b6f719ed76/onnxruntime/python/tools/transformers/dynamo_onnx_helper.py#L172
+    common_passes.ClearMetadataAndDocStringPass()(model)
```

Comment:
You may put all the passes into a pass manager like we do in optimize()

Reply:
Done